examples/analog_hamiltonian_simulation/04_Running_Analog_Hamiltonian_Simulation_with_local_simulator.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>We need to word this argument more carefully:</p><ul><li>"much longer time" -&gt; "long time" (there is nothing to compare against just yet)</li><li>"because of Rydberg blockade" -&gt; "because neighboring atoms are within each others Rydberg blockade radius ( ___ um)"</li><li>Add formula for computing the blockade radius: Rb = (C6 / |Delta_final|)^(1/6)</li><li>"Hence the effective Hamiltonian for adiabatic evolution is much smaller than the full Hamiltonian of the system" -&gt; "Hence we can approximate the full Hamiltonian of the system with a smaller one."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>